lmfdb/groups/abstract/circles.py

```diff
@@ -396,7 +396,7 @@ def pack(rdata, R0, rmax):
     # We approximate the intersections as happening at radius R0+rmax, then check that this doesn't cause problems
     rdata = sorted(rdata, key=lambda pair: (-pair[1].valuation(2), -pair[1].valuation(3), pair[1], -pair[0]))
     #print("Packing", R0, rmax, rdata)
-    radii = [r for (r, o) in rdata]
+    radii = [r for r, _ in rdata]
```

Comment:
Don't rename as underscore, for consistency with how we refer to the pairs in rdata.